src/sdk/object_sdk.js

```diff
@@ -181,6 +181,17 @@ class ObjectSDK {
         return bucket;
     }
 
+    async get_bucket_owner(bucket) {
+        // TODO: in the future we will add extra implementation per namespace
+        let bucket_info;
+        try {
+            bucket_info = await this.read_bucket_sdk_config_info(bucket);
+        } catch (err) {
+            if (err.rpc_code !== 'NO_SUCH_BUCKET') throw err;
+        }
+        return bucket_info;
```

Comment:
you need to return here the bucket_owner info and not the bucket info
```suggestion
        return {
              name: bucket_info.bucket_owner.unwrap(),
              id: bucket_info.owner_account.id,
          };
```

Reply:
In case of `namespace_fs`, the `id` is  not part of `bucket_info`. 

get_object_owner function  in `namespace_fs` 

https://github.com/noobaa/noobaa-core/pull/8062/files#diff-74a1674f40effb24da2459330c005cd5ab9fda4e1bca500a4387607926a2af1f

```
    async get_object_owner(object_sdk, bucket) {
        // TODO: in the future we will add extra implementation per namespace
        const info = await object_sdk.get_bucket_owner(bucket);
        return {
              name: info.bucket_owner.unwrap(),
              id: info.bucket_info.owner_account.id,  ================> this part (info.bucket_info.owner_account)
          };
    }
```


get_object_owner function in other namespace_***
https://github.com/noobaa/noobaa-core/pull/8062/files#diff-2243d9c761a2ed2a1a7b8a60ef26f68d78a66cc11851067db6cbe389279b23ea

```
    async get_object_owner(object_sdk, bucket) {
        // TODO: in the future we will add extra implementation per namespace
        const info = await object_sdk.get_bucket_owner(bucket);
        return {
              name: info.bucket_owner.unwrap(),
              id: info.owner_account.id,  =========================>  this part (info.ower_account)
          };
    }
```

So I thought to return general bucket_info from object_sdk and format it in namespacefs.js